internal/pkg/cli/interfaces.go

```diff
@@ -403,11 +403,15 @@ type deletePipelineRunner interface {
 	Run() error
 }
 
-type jobExecutor interface {
+type svcDeleteExecutor interface {
 	Execute() error
 }
 
-type askExecutor interface {
+type jobDeleteExecutor interface {
+	Execute() error
+}
+
+type envDeleteExecutor interface {
```

Comment:
These don't need to be separate interfaces, they have the same api:

```go
type executor interface {
	Execute() error
}

type executeAsker interface {
	Ask() error	
	executor
}
```
and then
```go
svcDeleteExecutor    func(svcName string) (executor, error)
```

Reply:
Done.